src/blocks/AppButtonsBlock.tsx

```diff
@@ -1,7 +1,23 @@
 import React from 'react'
 import { AppButtons } from 'components/AppButtons/AppButtons'
 import { ContentWrapper, SectionWrapper } from 'components/blockHelpers'
-import { BrandPivotBaseBlockProps } from './BaseBlockProps'
+import {
+  BrandPivotBaseBlockProps,
+  minimalColorComponentColors,
+} from './BaseBlockProps'
+
+type StandardColor = 'standard' | 'standard-inverse'
+
+const STANDARD_COLORS: Array<minimalColorComponentColors> = [
+  'standard',
+  'standard-inverse',
+]
+
+function isStandardColor(
+  color: minimalColorComponentColors,
+): color is StandardColor {
+  return STANDARD_COLORS.includes(color)
+}
```

Comment:
Might be just me that's too used to arrow functions :D But we try to stick to that for consistency or WDYT?

Reply:
Sure, I don't mind either way! #tabsVSspaces